src/components/MDXComponents.tsx

```diff
@@ -52,65 +31,56 @@ const MDXComponents = {
   // headings
   h1: ({ children }: any) => {
     return (
-      <Heading as='h1' textAlign='start' fontSize='4xl' mb={5}>
+      <Heading as='h1' textAlign='start' mb={5} {...header1}>
         {children}
       </Heading>
     );
   },
   h2: ({ children }: any) => {
     return (
-      <Heading as='h2' textAlign='start' fontSize='3xl' mb={4}>
+      <Heading as='h2' textAlign='start' mt='16 !important' mb={4} {...header2}>
         {children}
       </Heading>
     );
   },
   h3: ({ children }: any) => {
     return (
-      <Heading as='h3' fontSize='2xl' mt={5} mb={2.5}>
+      <Heading as='h3' mt={5} mb={2.5} {...header3}>
         {children}
       </Heading>
     );
   },
   h4: ({ children }: any) => {
     return (
-      <Heading as='h4' fontSize='lg' mb={2.5}>
+      <Heading as='h4' mb={2.5} {...header4}>
         {children}
       </Heading>
     );
   },
+  // tables
+  table: ({ children }: any) => (
+    <Flex maxW='min(100%, 100vw)' overflowX='scroll'>
+      <Table
+        variant='striped'
+        colorScheme='greenAlpha'
+        border='1px'
+        borderColor='blackAlpha.50'
+        my={6}
+        size={{ base: 'sm', lg: 'md' }}
+        w='auto'
+      >
+        {children}
+      </Table>
+    </Flex>
+  ),
   // pre
-  pre: ({ children }: any) => {
-    return (
-      <Stack mb={5}>
-        <pre>{children}</pre>
-      </Stack>
-    );
-  },
+  pre: ({ children }: any) => (
+    <Stack mb={5}>
+      <pre>{children}</pre>
+    </Stack>
+  ),
   // code
-  code: (code: any) => {
-    return (
-      <Code code={code} />
-    )
-
-  //   return !!code.inline ? (
-  //     <Text
-  //       as={'span'}
-  //       padding='0.125em 0.25em'
-  //       color='red.300'
-  //       background='code-bg-contrast'
-  //       borderRadius='0.25em'
-  //       fontFamily='code'
-  //       fontSize='sm'
-  //       overflowY='scroll'
-  //     >
-  //       {code.children[0]}
-  //     </Text>
-  //   ) : (
-  //     <Stack style={nightOwl}>
-  //       {code.children[0]}
-  //     </Stack>
-  //   );
-  }
+  code: ({ children, ...props }: any) => <Code {...props}>{children}</Code>
```

Comment:
```suggestion
  code: ({ children, ...props }: any) => <Code {...props}>{children}</Code>,
  // list
  ul: ({children}: any) => {
    return <UnorderedList mb={7} px={4}>{children}</UnorderedList>
  },
  li: ({ children }: any) => {
    return <ListItem color='primary'>{children}</ListItem>
  }
```

Missing some list styles here

Reply:
Good catch, thanks!